### PR TITLE
docs: improve descriptions for aws, azure and gcp integrations

### PR DESCRIPTION
**Pull request by WilliamEspegren**
# What is it?

- [ ] Feature / enhancement
- [ ] Bug
- [x] Docs / tests / types / typos

# Description

Simplified and clarified how to deploy qwik to Google Cloud Run, AWS lambda, Azure SWA.

# Checklist:

- [x] My code follows the [developer guidelines of this project](https://github.com/BuilderIO/qwik/blob/main/CONTRIBUTING.md)
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [ ] Added new tests to cover the fix / functionality

### CODE REVIEW

File: packages/docs/src/routes/docs/deployments/aws-lambda/index.mdx [hunk 1]
```diff
@@ -38,8 +38,8 @@ npm run build
 
 ## Deploy to AWS
 
-Before deploy is need setup your [AWS Credentials](https://www.serverless.com/framework/docs/providers/aws/guide/credentials) underhood
-the adapter  use [serverless](https://www.serverless.com/).
+Before deployment, you need to set up your [AWS Credentials](https://www.serverless.com/framework/docs/providers/aws/guide/credentials). Under the hood, 
+the adapter uses [serverless](https://www.serverless.com/).
```
Comment (gioboa) [inline packages/docs/src/routes/docs/deployments/aws-lambda/index.mdx:42]: You can add your name `- WilliamEspegren`in this file too 👍

Comment (WilliamEspegren) [reply packages/docs/src/routes/docs/deployments/aws-lambda/index.mdx:42]: Thanks that you saw it! Will do it right away 🙏

File: packages/docs/src/routes/docs/deployments/gcp-cloud-run/index.mdx [hunk 2]
```diff
@@ -33,12 +34,32 @@ After running `npm run build` you will have 2 build folders.
...
+
+   ```shell
+   gcloud auth login
+   ```
+
+   Grant the SDK access to the account you created in step 1.
+
+4. Change the name of the deploy script
+
+   Update the name of your Cloud Run app in the deploy script in your package.json.
```
Comment (gioboa) [inline packages/docs/src/routes/docs/deployments/gcp-cloud-run/index.mdx:57]: can you add the npm script to make it explicit?

Comment (gioboa) [inline packages/docs/src/routes/docs/deployments/gcp-cloud-run/index.mdx:57]:
```suggestion
   Update the name of your Cloud Run app in the deploy script in your package.json.
   `"deploy": "gcloud run deploy my-cloud-run-app --source .",`
  ```
```

Comment (gioboa) [reply packages/docs/src/routes/docs/deployments/gcp-cloud-run/index.mdx:57]: I added it

**Pull request merged**